docs/src/main/asciidoc/rest-client-reactive.adoc

```diff
@@ -708,6 +708,25 @@ You should see a JSON object containing some basic information about this extens
 As usual, the application can be packaged using `./mvnw clean package` and executed using the `target/quarkus-app/quarkus-run.jar` file.
 You can also generate the native executable with `./mvnw clean package -Pnative`.
 
+== Logging traffic
+REST Client Reactive can log the requests it sends and the responses it receives.
+To enable logging, add the `quarkus.rest-client.logging.scope` property to your `application.properties` and set it to:
+
+* `request-response` to log the request and response contents, or
+* `all` to also enable low level logging of the underlying libraries.
+
+As HTTP messages can have large bodies, we limit the amount of body characters logged. The default limit is `100`, you can change it by specifying `quarkus.rest-client.logging.body-limit`.
+
+An example logging configuration:
+
+[source,properties]
+----
+quarkus.rest-client.logging.scope=request-response
+quarkus.rest-client.logging.body-limit=50
+----
+
+NOTE: REST Client Reactive is logging the traffic with level DEBUG and does not alter logger properties. You may need to adjust your logger configuration to use this feature.
```

Comment:
Well this is not helping me, because I also need to figure out how to enable this logging. We should add that. Note that in https://github.com/quarkusio/quarkus/issues/22213 I suggested to "make it trivial to dump traffic" so I would ideally have preferred a one-line setting, but whatever. At least show the logging config to enable.

Also, I hope enabling logging doesn't then start logging all sorts of stuff that will hide the traffic?

Reply:
I added an example of logging setting. With it you get only sth like this in the log:
```
2021-12-22 07:51:35,218 DEBUG [org.jbo.res.rea.cli.log.DefaultClientLogger] (vert.x-eventloop-thread-47) Request: POST http://localhost:8081/hello Headers[Accept=text/plain;charset=UTF-8 Content-Type=application/json User-Agent=Resteasy Reactive Client content-length=5], Body:
Hello
2021-12-22 07:51:35,261 DEBUG [org.jbo.res.rea.cli.log.DefaultClientLogger] (vert.x-eventloop-thread-47) Redirect: POST http://localhost:8081/hello, Status[303 See Other], Headers[Content-Type=text/plain;charset=UTF-8 Location=http://example.com content-length=0], Body:

2021-12-22 07:51:35,471 DEBUG [org.jbo.res.rea.cli.log.DefaultClientLogger] (vert.x-eventloop-thread-47) Response: GET http://example.com/, Status[200 OK], Headers[Age=549036 Cache-Control=max-age=604800 Content-Type=text/html; charset=UTF-8 Date=Wed, 22 Dec 2021 06:51:35 GMT Etag="3147526947+ident" Expires=Wed, 29 Dec 2021 06:51:35 GMT Last-Modified=Thu, 17 Oct 2019 07:18:26 GMT Server=ECS (nyb/1D28) Vary=Accept-Encoding X-Cache=HIT content-length=1256], Body:
<!doctype html>
<html>
...
```

Reply:
I also moved the DefaultClientLogger class to a different package so that the property is a bit nicer.